### PR TITLE
Upload source distribution for Pypi and support conda build for bigdl-dllib

### PR DESCRIPTION
**Pull request by hkvision**
Conda support directly building conda packages from pypi packages using `conda skeleton` and `conda build`.
https://conda.io/projects/conda-build/en/latest/user-guide/tutorials/build-pkgs-skeleton.html

But the source (tar.gz file, platform independent) is needed to do this.
When installing from source, all the files needed in setup.py should be in the source package (i.e. scripts, version info)

This PR modifies the setup and release script to release the source distribution to pypi, builds the conda package of bigdl-dllib and installs to conda.

After bigdl-dllib is OK, we can do the same for other packages (under the condition that all the dependencies are available on conda).
TODO: add some documentation and description in conda page?

See the comments for steps and issues encountered and workarounds.

References:
https://blog.artwolf.in/a?ID=78598faa-bc1e-4d9b-8366-da5e3fccba63
https://docs.python.org/3/distutils/sourcedist.html
https://stackoverflow.com/questions/62568178/distribution-format-for-python-libraries-uploaded-on-pypi
Also referred to pyspark setup (for get version in setup.py).

### CODE REVIEW

**Comment by hkvision** [comment]
http://10.112.231.51:18888/job/BigDL2.0-NB-Deploy-PYTHON-PyPI-dllib-withsource/3/console

**Comment by hkvision** [comment]
```
conda skeleton pypi bigdl-dllib
conda build bigdl-dllib -c conda-forge
```

**Comment by hkvision** [comment]
```
(test-install) [root@Almaren-Node-188 kai]# conda build bigdl-dllib
No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.16
WARNING:conda_build.metadata:No numpy version specified in conda_build_config.yaml.  Falling back to default numpy value of 1.16
Adding in variants from internal_defaults
INFO:conda_build.variants:Adding in variants from internal_defaults
Attempting to finalize metadata for bigdl-dllib
INFO:conda_build.metadata:Attempting to finalize metadata for bigdl-dllib
Collecting package metadata (repodata.json): ...working... done
Solving environment: ...working... WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.*, but conda is ignoring the .* and treating it as 1
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 1.*, but conda is ignoring the .* and treating it as 1
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 2.*, but conda is ignoring the .* and treating it as 2
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 0.23.*, but conda is ignoring the .* and treating it as 0.23
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 0.23.*, but conda is ignoring the .* and treating it as 0.23
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.*, but conda is ignoring the .* and treating it as 3
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 4.*, but conda is ignoring the .* and treating it as 4
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 4.*, but conda is ignoring the .* and treating it as 4
WARNING:conda.models.version:Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.6.*, but conda is ignoring the .* and treating it as 3.6
WARNING conda.models.version:get_matcher(537): Using .* with relational operator is superfluous and deprecated and will be removed in a future version of conda. Your spec was 3.6.*, but conda is ignoring the .* and treating it as 3.6

Found conflicts! Looking for incompatible packages.
This can take several minutes.  Press CTRL-C to abort.
failed

Leaving build/test directories:
  Work:
 /root/anaconda3/conda-bld/work
  Test:
 /root/anaconda3/conda-bld/test_tmp
Leaving build/test environments:
  Test:
source activate  /root/anaconda3/conda-bld/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placeho
  Build:
source activate  /root/anaconda3/conda-bld/_build_env


Traceback (most recent call last):
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/environ.py", line 799, in get_install_actions
    actions = install_actions(prefix, index, specs, force=True)
  File "/root/anaconda3/lib/python3.8/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/root/anaconda3/lib/python3.8/site-packages/conda/plan.py", line 474, in install_actions
    txn = solver.solve_for_transaction(prune=prune, ignore_pinned=not pinned)
  File "/root/anaconda3/lib/python3.8/site-packages/conda/core/solve.py", line 114, in solve_for_transaction
    unlink_precs, link_precs = self.solve_for_diff(update_modifier, deps_modifier,
  File "/root/anaconda3/lib/python3.8/site-packages/conda/core/solve.py", line 157, in solve_for_diff
    final_precs = self.solve_final_state(update_modifier, deps_modifier, prune, ignore_pinned,
  File "/root/anaconda3/lib/python3.8/site-packages/conda/core/solve.py", line 281, in solve_final_state
    ssc = self._run_sat(ssc)
  File "/root/anaconda3/lib/python3.8/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/root/anaconda3/lib/python3.8/site-packages/conda/core/solve.py", line 815, in _run_sat
    ssc.solution_precs = ssc.r.solve(tuple(final_environment_specs),
  File "/root/anaconda3/lib/python3.8/site-packages/conda/common/io.py", line 88, in decorated
    return f(*args, **kwds)
  File "/root/anaconda3/lib/python3.8/site-packages/conda/resolve.py", line 1322, in solve
    self.find_conflicts(specs, specs_to_add, history_specs)
  File "/root/anaconda3/lib/python3.8/site-packages/conda/resolve.py", line 352, in find_conflicts
    raise UnsatisfiableError(bad_deps, strict=strict_channel_priority)
conda.exceptions.UnsatisfiableError: The following specifications were found to be incompatible with each other:

Output in format: Requested package -> Available versions

Package pypy3.7 conflicts for:
numpy[version='>=1.19.5'] -> python[version='>=3.7,<3.8.0a0'] -> pypy3.7[version='7.3.*|7.3.3.*|7.3.4.*|7.3.5.*|7.3.7.*']
numpy[version='>=1.19.5'] -> pypy3.7[version='>=7.3.3|>=7.3.4|>=7.3.5|>=7.3.7']
pip -> setuptools -> pypy3.7[version='7.3.3.*|7.3.4.*|7.3.5.*|7.3.7.*|>=7.3.3|>=7.3.5|>=7.3.7']
pyspark==2.4.6 -> numpy[version='>=1.7'] -> pypy3.7[version='7.3.3.*|7.3.4.*|7.3.5.*|7.3.7.*|>=7.3.3|>=7.3.4|>=7.3.5|>=7.3.7']
six[version='>=1.10.0'] -> python -> pypy3.7[version='7.3.3.*|7.3.4.*|7.3.5.*|7.3.7.*']

Package wheel conflicts for:
python=3.8 -> pip -> wheel
pip -> wheel

Package setuptools conflicts for:
python=3.8 -> pip -> setuptools
pyspark==2.4.6 -> pandas[version='>=0.19.2'] -> setuptools[version='<60.0.0']
pip -> setuptools

Package numpy conflicts for:
pyspark==2.4.6 -> numpy[version='>=1.7']
pyspark==2.4.6 -> pandas[version='>=0.19.2'] -> numpy[version='1.10.*|1.11.*|1.12.*|1.13.*|>=1.11|>=1.11.*|>=1.11.3,<2.0a0|>=1.12.1,<2.0a0|>=1.13.3,<2.0a0|>=1.14.6,<2.0a0|>=1.15.4,<2.0a0|>=1.16.6,<2.0a0|>=1.19.2,<2.0a0|>=1.20.3,<2.0a0|>=1.20.2,<2.0a0|>=1.9.3,<2.0a0|>=1.9|>=1.21.5,<2.0a0|>=1.19.5,<2.0a0|>=1.18.5,<2.0a0|>=1.21.4,<2.0a0|>=1.17.5,<2.0a0|>=1.19.4,<2.0a0|>=1.16.5,<2.0a0|>=1.18.4,<2.0a0|>=1.18.1,<2.0a0|>=1.9.*|>=1.8']
numpy[version='>=1.19.5']

Package six conflicts for:
six[version='>=1.10.0']
numpy[version='>=1.19.5'] -> mkl-service[version='>=2.3.0,<3.0a0'] -> six

Package tzdata conflicts for:
pip -> python[version='>=3.10,<3.11.0a0'] -> tzdata
numpy[version='>=1.19.5'] -> python[version='>=3.10,<3.11.0a0'] -> tzdata
six[version='>=1.10.0'] -> python -> tzdata

Package ca-certificates conflicts for:
pip -> python[version='>=2.7,<2.8.0a0'] -> ca-certificates
python=3.8 -> openssl[version='>=1.1.1l,<1.1.2a'] -> ca-certificates
pyspark==2.4.6 -> python[version='<3.8'] -> ca-certificates
six[version='>=1.10.0'] -> python -> ca-certificates

Package pip conflicts for:
pyspark==2.4.6 -> python[version='<3.8'] -> pip
six[version='>=1.10.0'] -> python -> pip
numpy[version='>=1.19.5'] -> python[version='>=3.10,<3.11.0a0'] -> pip
python=3.8 -> pip

Package _libgcc_mutex conflicts for:
python=3.8 -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main']
numpy[version='>=1.19.5'] -> libgcc-ng[version='>=7.5.0'] -> _libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main']The following specifications were found to be incompatible with your system:

  - feature:/linux-64::__glibc==2.17=0
  - feature:|@/linux-64::__glibc==2.17=0
  - numpy[version='>=1.19.5'] -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']
  - python=3.8 -> libgcc-ng[version='>=7.5.0'] -> __glibc[version='>=2.17']

Your installed version is: 2.17


During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/root/anaconda3/bin/conda-build", line 11, in <module>
    sys.exit(main())
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/cli/main_build.py", line 480, in main
    execute(sys.argv[1:])
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/cli/main_build.py", line 469, in execute
    outputs = api.build(args.recipe, post=args.post, test_run_post=args.test_run_post,
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/api.py", line 186, in build
    return build_tree(
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/build.py", line 3083, in build_tree
    packages_from_this = build(metadata, stats,
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/build.py", line 2046, in build
    output_metas = expand_outputs([(m, need_source_download, need_reparse_in_env)])
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/render.py", line 788, in expand_outputs
    for (output_dict, m) in deepcopy(_m).get_output_metadata_set(permit_unsatisfiable_variants=False):
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/metadata.py", line 2120, in get_output_metadata_set
    conda_packages = finalize_outputs_pass(ref_metadata, conda_packages, pass_no=0,
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/metadata.py", line 781, in finalize_outputs_pass
    fm = finalize_metadata(om, parent_metadata=parent_metadata,
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/render.py", line 546, in finalize_metadata
    build_unsat, host_unsat = add_upstream_pins(m,
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/render.py", line 408, in add_upstream_pins
    host_deps, host_unsat, extra_run_specs_from_host = _read_upstream_pin_files(m, 'host',
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/render.py", line 373, in _read_upstream_pin_files
    deps, actions, unsat = get_env_dependencies(m, env, m.config.variant,
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/render.py", line 138, in get_env_dependencies
    actions = environ.get_install_actions(tmpdir, tuple(dependencies), env,
  File "/root/anaconda3/lib/python3.8/site-packages/conda_build/environ.py", line 801, in get_install_actions
    raise DependencyNeedsBuildingError(exc, subdir=subdir)
conda_build.exceptions.DependencyNeedsBuildingError: Unsatisfiable dependencies for platform linux-64: {"pypy3.7[version='>=7.3.3|>=7.3.4|>=7.3.5|>=7.3.7']", 'tzdata', 'feature:|@/linux-64::__glibc==2.17=0', "pypy3.7[version='7.3.3.*|7.3.4.*|7.3.5.*|7.3.7.*|>=7.3.3|>=7.3.5|>=7.3.7']", "pypy3.7[version='7.3.*|7.3.3.*|7.3.4.*|7.3.5.*|7.3.7.*']", 'wheel', 'setuptools', 'ca-certificates', "pypy3.7[version='7.3.3.*|7.3.4.*|7.3.5.*|7.3.7.*|>=7.3.3|>=7.3.4|>=7.3.5|>=7.3.7']", "numpy[version='>=1.7']", 'pip', "setuptools[version='<60.0.0']", 'six', "__glibc[version='>=2.17']", "pypy3.7[version='7.3.3.*|7.3.4.*|7.3.5.*|7.3.7.*']", 'feature:/linux-64::__glibc==2.17=0', "numpy[version='1.10.*|1.11.*|1.12.*|1.13.*|>=1.11|>=1.11.*|>=1.11.3,<2.0a0|>=1.12.1,<2.0a0|>=1.13.3,<2.0a0|>=1.14.6,<2.0a0|>=1.15.4,<2.0a0|>=1.16.6,<2.0a0|>=1.19.2,<2.0a0|>=1.20.3,<2.0a0|>=1.20.2,<2.0a0|>=1.9.3,<2.0a0|>=1.9|>=1.21.5,<2.0a0|>=1.19.5,<2.0a0|>=1.18.5,<2.0a0|>=1.21.4,<2.0a0|>=1.17.5,<2.0a0|>=1.19.4,<2.0a0|>=1.16.5,<2.0a0|>=1.18.4,<2.0a0|>=1.18.1,<2.0a0|>=1.9.*|>=1.8']", "_libgcc_mutex[version='*|0.1|0.1',build='conda_forge|main']"}
```

**Comment by hkvision** [comment]
```
  File "/root/anaconda3/envs/conda-install/conda-bld/bigdl-dllib_1644914832361/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.7/site-packages/bigdl/dllib/keras/layers/__init__.py", line 30, in <module>
import: 'bigdl.dllib.feature.transform.vision'
import: 'bigdl.dllib.keras'
import: 'bigdl.dllib.keras.datasets'
import: 'bigdl.dllib.keras.engine'
import: 'bigdl.dllib.keras.layers'
    from .self_attention import *
  File "/root/anaconda3/envs/conda-install/conda-bld/bigdl-dllib_1644914832361/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.7/site-packages/bigdl/dllib/keras/layers/self_attention.py", line 28, in <module>
    from bigdl.dllib.keras.models import Sequential
  File "/root/anaconda3/envs/conda-install/conda-bld/bigdl-dllib_1644914832361/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.7/site-packages/bigdl/dllib/keras/models.py", line 23, in <module>
    from bigdl.dllib.utils.bigdl_export import keras_export
  File "/root/anaconda3/envs/conda-install/conda-bld/bigdl-dllib_1644914832361/_test_env_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_placehold_pl/lib/python3.7/site-packages/bigdl/dllib/utils/bigdl_export.py", line 23, in <module>
    from tensorflow.python.util import tf_decorator
ModuleNotFoundError: No module named 'tensorflow'
Tests failed for bigdl-dllib-0.14.0b20220215003-py37_0.tar.bz2 - moving package to /root/anaconda3/envs/conda-install/conda-bld/broken
WARNING:conda_build.build:Tests failed for bigdl-dllib-0.14.0b20220215003-py37_0.tar.bz2 - moving package to /root/anaconda3/envs/conda-install/conda-bld/broken
WARNING conda_build.build:tests_failed(2971): Tests failed for bigdl-dllib-0.14.0b20220215003-py37_0.tar.bz2 - moving package to /root/anaconda3/envs/conda-install/conda-bld/broken
TESTS FAILED: bigdl-dllib-0.14.0b20220215003-py37_0.tar.bz2
```

File: python/dllib/dev/release/release.sh [hunk 1]
```diff
@@ -85,12 +82,21 @@ if [ -d "${BIGDL_DIR}/python/dllib/src/bigdl_dllib.egg-info" ]; then
...
+echo "Packing python source code and distribution: $wheel_command"
 ${wheel_command}
 
 if [ ${upload} == true ]; then
     upload_command="twine upload dist/bigdl_dllib-${bigdl_version}-py3-none-${verbose_pname}.whl"
     echo "Please manually upload with this command: $upload_command"
     $upload_command
+    upload_command2="twine upload dist/bigdl-dllib-${bigdl_version}.tar.gz"
+    echo "Please manually upload with this command: $upload_command2"
+    $upload_command2
```
Comment (hkvision) [inline python/dllib/dev/release/release.sh:101]: modify this to upload only for linux before the merge.

**Comment by hkvision** [comment]
@jason-dai @yangw1234 @glorysdj Take a look at the above steps and comments to build for conda.

**Comment by hkvision** [comment]
https://anaconda.org/hkvision/bigdl-dllib
Uploaded to my own account and the installation is successful:
```
https://anaconda.org/hkvision/bigdl-dllib
```

```
(dllib-conda) [root@clx006 ~]# conda install -c hkvision bigdl-dllib
Collecting package metadata (current_repodata.json): done
Solving environment: failed with initial frozen solve. Retrying with flexible solve.
Solving environment: failed with repodata from current_repodata.json, will retry with next repodata source.
Collecting package metadata (repodata.json): done
Solving environment: done

## Package Plan ##

  environment location: /opt/work/anaconda3/envs/dllib-conda

  added / updated specs:
    - bigdl-dllib


The following packages will be downloaded:

    package                    |            build
    ---------------------------|-----------------
    bigdl-dllib-0.14.0b20220215004|           py37_0        96.3 MB  hkvision
    blas-1.0                   |              mkl           6 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    bottleneck-1.3.2           |   py37heb32a55_1         125 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    intel-openmp-2021.4.0      |    h06a4308_3561         4.2 MB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    mkl-2021.4.0               |     h06a4308_640       142.6 MB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    mkl-service-2.4.0          |   py37h7f8727e_0          56 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    mkl_fft-1.3.1              |   py37hd3c417c_0         172 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    mkl_random-1.2.2           |   py37h51133e4_0         287 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    numexpr-2.8.1              |   py37h6abb31d_0         123 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    numpy-1.21.2               |   py37h20f2e39_0          23 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    numpy-base-1.21.2          |   py37h79a1101_0         4.8 MB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    packaging-21.3             |     pyhd3eb1b0_0          36 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    pandas-1.3.5               |   py37h8c16a72_0         9.3 MB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    py4j-0.10.7                |           py37_0         241 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    pyparsing-3.0.4            |     pyhd3eb1b0_0          81 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    pyspark-2.4.6              |             py_0       207.6 MB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/cloud/conda-forge
    python-dateutil-2.8.2      |     pyhd3eb1b0_0         233 KB  https://mirrors.tuna.tsinghua.edu.cn/anaconda/pkgs/main
    ------------------------------------------------------------
                                           Total:       466.1 MB

The following NEW packages will be INSTALLED:

  bigdl-dllib        hkvision/linux-64::bigdl-dllib-0.14.0b20220215004-py37_0
  blas               anaconda/pkgs/main/linux-64::blas-1.0-mkl
  bottleneck         anaconda/pkgs/main/linux-64::bottleneck-1.3.2-py37heb32a55_1
  intel-openmp       anaconda/pkgs/main/linux-64::intel-openmp-2021.4.0-h06a4308_3561
  mkl                anaconda/pkgs/main/linux-64::mkl-2021.4.0-h06a4308_640
  mkl-service        anaconda/pkgs/main/linux-64::mkl-service-2.4.0-py37h7f8727e_0
  mkl_fft            anaconda/pkgs/main/linux-64::mkl_fft-1.3.1-py37hd3c417c_0
  mkl_random         anaconda/pkgs/main/linux-64::mkl_random-1.2.2-py37h51133e4_0
  numexpr            anaconda/pkgs/main/linux-64::numexpr-2.8.1-py37h6abb31d_0
  numpy              anaconda/pkgs/main/linux-64::numpy-1.21.2-py37h20f2e39_0
  numpy-base         anaconda/pkgs/main/linux-64::numpy-base-1.21.2-py37h79a1101_0
  packaging          anaconda/pkgs/main/noarch::packaging-21.3-pyhd3eb1b0_0
  pandas             anaconda/pkgs/main/linux-64::pandas-1.3.5-py37h8c16a72_0
  py4j               anaconda/pkgs/main/linux-64::py4j-0.10.7-py37_0
  pyparsing          anaconda/pkgs/main/noarch::pyparsing-3.0.4-pyhd3eb1b0_0
  pyspark            anaconda/cloud/conda-forge/noarch::pyspark-2.4.6-py_0
  python-dateutil    anaconda/pkgs/main/noarch::python-dateutil-2.8.2-pyhd3eb1b0_0
  pytz               anaconda/pkgs/main/noarch::pytz-2021.3-pyhd3eb1b0_0
  six                anaconda/pkgs/main/noarch::six-1.16.0-pyhd3eb1b0_1


Proceed ([y]/n)? y


Downloading and Extracting Packages
packaging-21.3       | 36 KB     | ################################################################################################################### | 100%
bottleneck-1.3.2     | 125 KB    | ################################################################################################################### | 100%
mkl-2021.4.0         | 142.6 MB  | ################################################################################################################### | 100%
pandas-1.3.5         | 9.3 MB    | ################################################################################################################### | 100%
mkl_random-1.2.2     | 287 KB    | ################################################################################################################### | 100%
numpy-base-1.21.2    | 4.8 MB    | ################################################################################################################### | 100%
intel-openmp-2021.4. | 4.2 MB    | ################################################################################################################### | 100%
pyparsing-3.0.4      | 81 KB     | ################################################################################################################### | 100%
numpy-1.21.2         | 23 KB     | ################################################################################################################### | 100%
mkl_fft-1.3.1        | 172 KB    | ################################################################################################################### | 100%
mkl-service-2.4.0    | 56 KB     | ################################################################################################################### | 100%
py4j-0.10.7          | 241 KB    | ################################################################################################################### | 100%
python-dateutil-2.8. | 233 KB    | ################################################################################################################### | 100%
numexpr-2.8.1        | 123 KB    | ################################################################################################################### | 100%
blas-1.0             | 6 KB      | ################################################################################################################### | 100%
pyspark-2.4.6        | 207.6 MB  | ################################################################################################################### | 100%
bigdl-dllib-0.14.0b2 | 96.3 MB   | ################################################################################################################### | 100%
Preparing transaction: done
Verifying transaction: done
Executing transaction: done
```

File: python/dllib/src/bigdl/dllib/utils/bigdl_export.py [hunk 2]
```diff
@@ -51,8 +49,12 @@ def __init__(self, *args, **kwargs):  # pylint: disable=g-doc-args
     def __call__(self, func):
         api_names_attr = API_ATTRS[self._api_name].names
 
-        _, undecorated_func = tf_decorator.unwrap(func)
-        self.set_attr(undecorated_func, api_names_attr, self._names)
+        try:  # temporary solution
```
Comment (jason-dai) [inline python/dllib/src/bigdl/dllib/utils/bigdl_export.py:52]: @dding3 is your fix ready for merge?

**Comment by hkvision** [comment]
jenkins passed: http://10.112.231.51:18888/view/BigDL-2.0-PR/job/BigDL-PR-Dllib-Validation/318/
tested manually. Merging it first. Tell me if there are any issues related to the changes.

**Pull request merged**